src/components/v5/common/ActionSidebar/partials/forms/SimplePaymentForm/SimplePaymentForm.tsx

```diff
@@ -18,7 +18,7 @@ const displayName = 'v5.common.ActionSidebar.partials.SimplePaymentForm';
 const SimplePaymentForm: FC<ActionFormBaseProps> = ({ getFormOptions }) => {
   const { decisionMethods } = useDecisionMethods();
 
-  useSimplePayment(getFormOptions);
+  const { selectedTokenAddress } = useSimplePayment(getFormOptions);
```

Comment:
As mentioned during our chat in Discord, I don't think it's necessary to pass the token address explicitly, as the `AmountField` can access it directly from the form state.